examples/quiz/en.yaml

```diff
@@ -0,0 +1,43 @@
+schema_version: 1
+
+locale: en
```

Comment:
Having one locale per .yml file is cool for the upload interface and the GitWhisperer implementation, but it's contrary to the content bundle design specs.

Check out the example lab's use of default locale:

https://github.com/CloudVLab/qwiklabs-content-bundle-spec/blob/3cdcd1909acff93a1617836273e89612dfcb40bd/examples/lab-robust/qwiklabs.yaml#L5

... and localizable properties:

https://github.com/CloudVLab/qwiklabs-content-bundle-spec/blob/3cdcd1909acff93a1617836273e89612dfcb40bd/examples/lab-robust/qwiklabs.yaml#L7-L15

Reply:
@rickcnagy - poke